### DISCUSSION TITLE
push sprite to another sprite

### DISCUSSION THREAD

**Discussion by Nickbuster** [Bodmer/TFT_eSPI#2509] [Q&A - Sprites] [answered]
I have been trying to push one sprite i/onto another but I do not see the result.
Example code:

`spriteBack.setSwapBytes(true);
  spriteBack.setColorDepth(16);
  spriteBack.createSprite(240, 240);
  spriteBack.fillSprite(TFT_BLUE);

  spriteFront.setSwapBytes(true);
  spriteFront.setColorDepth(16);
  spriteFront.createSprite(240, 240);
  spriteFront.fillSprite(TFT_GREEN);
  
  spriteFront.pushToSprite(&spriteBack, 0, 0);
  
  spriteBack.pushSprite(0, 0);`
  
  This code should result green screen display, yet display is filled with blue color.

**Comment by Bodmer** [comment, marked as answer]
I suspect that there is insufficient memory for the second sprite. Try with smaller sprites.

**Reply by Nickbuster** [reply]
Thank you and thank you for the library!
